aesara/tensor/math_opt.py

```diff
@@ -3009,6 +3009,8 @@ def _is_1(expr):
     allow_multiple_clients=True,
     values_eq_approx=values_eq_approx_remove_inf,
     skip_identities_fn=_skip_mul_1,
+    tracks=[sigmoid],
```

Comment:
Again, we don't want to use specific `Op` instances as `tracks` arguments, unless we need to for some reason.

Reply:
See my comment on the general thread. PatternSub always uses tracks under the hood